test/integration/freetext_editor_spec.mjs

```diff
@@ -1364,9 +1361,6 @@ describe("FreeText Editor", () => {
           // Enter in editing mode.
           await switchToFreeText(page);
 
-          // eslint-disable-next-line no-restricted-syntax
-          await waitForTimeout(200);
```

Comment:
Note that `switchToFreeText` waits for the switching to be complete, see https://github.com/mozilla/pdf.js/blob/master/test/integration/freetext_editor_spec.mjs#L84, so this occurrence can therefore be removed without replacement.